package.json

```diff
@@ -3,9 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.10.7",
+    "@apollo/client": "^3.8.7",
```

Comment:
Was this intentional? Might have been a `main` merge perhaps, but I think we should be able to keep 3.10.7 unless that upgrade specifically broke something

Reply:
Gotcha probably yeah